### PR TITLE
Fix renaming battle rooms

### PR DESCRIPTION
**Pull request by AnnikaCodes**
This prevents "while searching, room ... expired" errors.

### CODE REVIEW

File: server/chat-commands/core.ts [hunk 1]
```diff
@@ -370,7 +370,7 @@ export const commands: ChatCommands = {
 		if (this.cmd === 'mee' && /[A-Z-a-z0-9/]/.test(target.charAt(0))) {
 			return this.errorReply(this.tr`/mee - must not start with a letter or number`);
 		}
-		this.checkChat(`/${this.cmd} ${target || ''}`);
+		target = this.checkChat(`/${this.cmd} ${target || ''}`);
```
Comment (Zarel) [inline server/chat-commands/core.ts:373]: This is probably a mistake.

**Comment by Zarel** [comment]
The rest looks good and is ready for the next restart.

File: server/chat-commands/core.ts (hunk 1 above)
Comment (AnnikaCodes) [reply server/chat-commands/core.ts]: I don't think so? Without this, `/me` doesn't work.

Comment (mia-pi-git) [reply server/chat-commands/core.ts]: I'm fixing this in #7327. You can remove it.

**Review by Zarel — APPROVED** [review]

**Pull request merged**